topics/introduction/tutorials/vsi_qc/tutorial.md

```diff
@@ -0,0 +1,244 @@
+@M02286:19:000000000-AA549:1:1101:11116:1322 1:N:0:23
+CCTACGGGAGGCAGCAGTAGGGAATCTTCGGCAATGGACGGAAGTCTGACCGAGCAACGCCGCGTGAGT
++
+AAC<CCF+@@>CC,C9,F9C9@9-CFFFE@7@:+CC8-C@:7,@EFE,6CF:+8F7EFEEF@EGGGEEE
+
+```
+
+Each sequencing read is represented by four lines:
+
+1. `@` followed by read ID and optional information about sequencing run
+2. sequenced bases
+3. `+` (optionally followed by the read ID and some additional info)
+4. quality scores for each base of the sequence encoded as [ASCII symbols](https://en.wikipedia.org/wiki/ASCII)
+
+## Paired end data
+
+It is common to prepare pair-end and mate-pair sequencing libraries. This is highly beneficial for a number of applications discussed in subsequent topics. For now let's just briefly discuss what these are and how they manifest themselves in FASTQ form.
+
+![Paired-end and mate-pair reads](../../images/pe_mp.png "<b>Paired-end and mate-pair reads</b>. In paired end sequencing (left) the actual ends of rather short DNA molecules (less than 1kb) are determined, while for mate pair sequencing (right) the ends of long molecules are joined and prepared in special sequencing libraries. In these mate pair protocols, the ends of long, size-selected molecules are connected with an internal adapter sequence (i.e. linker, yellow) in a circularization reaction. The circular molecule is then processed using restriction enzymes or fragmentation. Fragments are enriched for the linker and outer library adapters are added around the two combined molecule ends. The internal adapter can then be used as a second priming site for an additional sequencing reaction in the same orientation or sequencing can be performed from the second adapter, from the reverse strand. (From <i>Understanding and improving high-throughput sequencing data production and analysis</i>, Ph.D. dissertation by <a href='https://ul.qucosa.de/api/qucosa%3A11231/attachment/ATT-0/'>Martin Kircher</a>)")
+
+
+Thus in both cases (paired-end and mate-pair) a single physical piece of DNA (or RNA in the case of RNA-seq) is sequenced from two ends and so generates two reads. These can be represented as separate files (two FASTQ files with first and second reads) or a single file were reads for each end are interleaved. Here are examples:
+
+### Two single files
+
+File 1
+
+```
+ @M02286:19:000000000-AA549:1:1101:12677:1273 1:N:0:23
+ CCTACGGGTGGCAGCAGTGAGGAATATTGGTCAATGGACGGAAGTCT
+ +
+ ABC8C,:@F:CE8,B-,C,-6-9-C,CE9-CC--C-<-C++,,+;CE
+ @M02286:19:000000000-AA549:1:1101:15048:1299 1:N:0:23
+ CCTACGGGTGGCTGCAGTGAGGAATATTGGACAATGGTCGGAAGACT
+ +
+ ABC@CC77CFCEG;F9<F89<9--C,CE,--C-6C-,CE:++7:,CF
+```
+
+File 2
+
+```
+@M02286:19:000000000-AA549:1:1101:12677:1273 2:N:0:23
+CACTACCCGTGTATCTAATCCTGTTTGATACCCGCACCTTCGAGCTTA
++
+--8A,CCE+,,;,<CC,,<CE@,CFD,,C,CFF+@+@CCEF,,,B+C,
+@M02286:19:000000000-AA549:1:1101:15048:1299 2:N:0:23
+CACTACCGGGGTATCTAATCCTGTTCGCTCCCCACGCTTTCGTCCATC
++
+-6AC,EE@::CF7CFF<<FFGGDFFF,@FGGGG?F7FEGGGDEFF>FF
+```
```

Comment:
if you want a side-by-side rendering, [consider using the `code-in` and `code-out` boxes. ](https://training.galaxyproject.org/training-material/topics/contributing/tutorials/create-new-tutorial-content/tutorial.html#code-box)